html/settings/_colors.scss

```diff
@@ -4,29 +4,108 @@
 
 // Primary
 /// Color value for red.
-/// @group red
+/// @group primary
 /// @type #C8102E
 $sprk-red: rgb(200, 16, 46);
-/// Color value for mid red.
-/// @group red
-/// @type #9D2235
-$sprk-mid-red: rgb(157,34,53);
 /// Color value for deep red.
-/// @group red
+/// @group primary
 /// @type #76232F
-$sprk-deep-red: rgb(118,35,47);
-/// Color value for black.
-/// @group black
-/// @type #333333
-$sprk-black: rgb(51, 51, 51);
+$sprk-deep-red: rgb(118, 35, 47);
+/// Color value for mid red.
+/// @group primary
+/// @type #9D2235
+$sprk-mid-red: rgb(157, 34, 53);
+
+// Secondary
+/// Color value for purple.
+/// @group secondary
+/// @type #603AA1
+$sprk-purple: rgb(96, 58, 161);
+/// Color value for dark purple.
+/// @group secondary
+/// @type #3C3174
+$sprk-dark-purple: rgb(60, 49, 116);
+/// Color value for deep purple.
+/// @group secondary
+/// @type #2E1166
+$sprk-deep-purple: rgb(46, 17, 102);
+/// Color value for light purple.
+/// @group secondary
+/// @type #9265D3
+$sprk-light-purple: rgb(146, 101, 211);
+/// Color value for lightest purple.
+/// @group secondary
+/// @type #E7E2F2
+$sprk-lightest-purple: rgb(231, 226, 242);
```

Comment:
lightest? haha, is this what design is calling it? I like before how we had color, mid-color, etc.

Reply:
@afebbraro This is the naming I got from Jane. There's gonna be a lot more shades added too. So there will end up being a mid and everything too. :woman_shrugging: